crates/cli/tests/cli_run.rs

```diff
@@ -1179,25 +1179,25 @@ mod cli_run {
         #[test]
         fn rbtree_del() {
             test_benchmark(
-                "RBTreeDel.roc",
+                "rBTreeDel.roc",
                 &["420"],
-                &[],
                 "30\n",
-                true
+                UseValgrind::Yes,
```

Comment:
This was still using the old format. I tried to re-enable this test, but I spent a lot of time updating rBTreeDel.roc to current Roc and gave up.